docs/pages/reference/yaml-reference.md

```diff
@@ -50,7 +54,11 @@ This documentation effectively reflects the Java object tree under [RawServiceSp
 
   * `count`
 
-    The number of pods of this type to be deployed. This may either be hardcoded or exposed to end users via mustache templating.
+    The number of pods of this type to be deployed. This may either be hardcoded or exposed to end users via mustache templating. This value may be always increased after the service has been deployed, but it can only be decreased if `allow-decomission` is `true`.
+
+  * `allow-decommission`
+
+    Whether to allow the `count` to be decreased by an operator in a configuration update. For safety reasons this defaults to `false`, but the service developer may set this field to `true` to explicitly allow scale-down on a per-pod basis.
```

Comment:
"the pod `count`" or just "~the~ `count`"?

Reply:
Went with "this pod's `count`"